kloppy/domain/models/common.py

```diff
@@ -294,6 +302,79 @@ def duration(self):
         return self.end_timestamp - self.start_timestamp
 
 
+class Origin(Enum):
+    """
+    Attributes:
+        TOP_LEFT: Origin at the top left of the field
+        BOTTOM_RIGHT: Origin at the bottom left of the field
+        CENTER: Origin at the center of the field
+    """
+
+    TOP_LEFT = "top-left"
+    BOTTOM_RIGHT = "bottom-right"
+    CENTER = "center"
+
+    def __str__(self):
+        return self.value
+
+
+@dataclass
+class CoordinateSystem(ABC):
+    provider: Provider
+    origin: Origin
```

Comment:
@koenvo The `origin` attribute is not actually needed because it's implicit on the `dimension`, but I think is better to keep it because it's transparent, descriptive and could be useful down the line.

Reply:
Totally agree